code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1782,31 +1748,61 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 				H.adjustBruteLoss(LOW_PRESSURE_DAMAGE * H.physiology.pressure_mod)
 				H.throw_alert("pressure", /obj/screen/alert/lowpressure, 2)
 
-/// Used to stabilize the body temperature back to normal on living mobs
-/// Returns the amount of degrees kelvin to change the body temperature
-/datum/species/proc/natural_bodytemperature_stabilization(mob/living/carbon/human/H)
+/**
+ * Used to stabilize the body temperature back to normal on living mobs
+ *
+ * vars:
+ * * environment The environment gas mix
+ * * H The mob we will stabilize
+ */
+/datum/species/proc/natural_bodytemperature_stabilization(datum/gas_mixture/environment, mob/living/carbon/human/H)
```

Comment:
is this a duplicate of the carbon still? they look awfully similar

Reply:
This version uses the species body temperature customizations.

Reply:
rad